cmd/glasskube/cmd/update.go

```diff
@@ -93,6 +99,41 @@ func printTransaction(tx update.UpdateTransaction) {
 	_ = w.Flush()
 }
 
+func handleOutput(pkgs []v1alpha1.Package) {
+	if updateCmdOptions.Output == "" {
+		return
+	}
+
+	var outputData []byte
+	var err error
+	switch updateCmdOptions.Output {
+	case OutputFormatJSON:
+		outputData, err = json.Marshal(pkgs)
+	case OutputFormatYAML:
+		var buffer bytes.Buffer
+		for _, pkg := range pkgs {
+			data, err := yaml.Marshal(pkg)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "❌ failed to marshal output: %v\n", err)
+				cliutils.ExitWithError()
+			}
+			buffer.Write(data)
+			buffer.Write([]byte("---\n"))
```

Comment:
This leads to an empty document at the end. Please print the document separator before the content and only if there are two or more packages.